reviewers.js

```diff
@@ -56,4 +68,15 @@ function getReviewerByReviewTitle(reviewTitle, reviewers) {
 function searchReviewers(query, reviewers) {
```

Comment:
You can not use `return` **three** times in the same function, the `return` should be only use once, unless it's inside an `if` statement or a `swtich` case.

This function could be easily done using the `filter` method:
```
 return reviewers.filter(
    (reviewer) =>
      reviewer.reviewerName.includes(query) ||
      reviewer.description.includes(query)
  );
```